code/server/test/api_endpoint_tests.js

```diff
@@ -1,26 +1,29 @@
 var app = require('../main.js');
 var request = require('supertest');
 var assert = require('assertthat'); // View README for documentation https://github.com/thenativeweb/assertthat
-
+var api = require('../routes/api.js');
+var stubUser = require('../stub_models/StubUser.js');
+var stubLikes = require('../stub_models/StubUserMatches.js');
 //////////////////////////////////////////
 // ALL API TESTS SHOULD GO IN THIS FILE // 
 //////////////////////////////////////////
 
 describe('api', function() {
+          api.injectUser(stubUser);
+          api.injectLikes(stubLikes);
```

Comment:
I wasn't sure I could go api.inject or app.api.inject with only the main import.  One thing I noticed is that the changes are carrying over.  In my user unit tests, I had changed some values in the stub ('steve' to 'Steve'), then tested for 'steve' in the api tests, but it couldn't verify 'steve', since the unit tests happened to run first and the value was 'Steve'.

tl:dr - Not sure why it works, but it does.  I'll test using just main.